source/Sashimi.Aws.Tests/CloudFormation/package-withparameters/parameters.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "ParameterKey": "NameVarParam",
+    "ParameterValue": "#{NameVarParamValue}"
+  },
+  {
+    "ParameterKey": "NamePlainParam",
+    "ParameterValue": "#{NamePlainParamValue}"
```

Comment:
I would recommend using a different "syntax" for values that we are substituting in the test setup vs. values that are substituted by Octopus, like `@NamePlainParamValue`.